build.gradle.kts

```diff
@@ -38,7 +38,7 @@ val dokkaJar = tasks.register<Jar>("dokkaJar") {
     archiveClassifier.set("javadoc")
 }
 
-version = "2.3.0-SNAPSHOT"
+version = "2.4.0-SNAPSHOT"
```

Comment:
Please don't do this in this PR.
The PR is only here to fix the visibility.
Releases will happen somewhere else.